plrustc/plrustc/src/main.rs

```diff
@@ -95,6 +196,63 @@ fn arg_value<'a, T: AsRef<str>>(args: &'a [T], find_arg: &str) -> Option<&'a str
     None
 }
 
+struct ErrorHidingFileLoader;
+
+fn replacement_error() -> std::io::Error {
+    // Unix-ism, but non-unix could use `io::Error::from(ErrorKind::NotFound)`
```

Comment:
Let's just say Postgres itself has a lot of portability shims aaand well, it tends to shim other systems into looking more like Unix rather than the other way around.